src/bindings.h

```diff
@@ -57,6 +58,16 @@ enum lxcfs_virt_t {
 	LXC_TYPE_PROC_SLABINFO,
 #define LXC_TYPE_PROC_SLABINFO_PATH "/proc/slabinfo"
 
+	LXC_TYPE_PROC_PRESSURE,
```

Comment:
I don't think that you can add new items in the middle of `enum` as it can (and will) break liblxc live upgrades.

Reply:
well, if add at the end, the LXCFS_TYPE_* macros will become really complicated,
i will think about a better way to deal with it.

Reply:
It become complicated after commit cf02ccbe, when new proc type is added, we have to change the macro.
Maybe revert to original way is better. I rebased the code and force pushed, please kindly review it again.
Thanks.